tests/sources/test_azure_blob_storage.py

```diff
@@ -465,7 +465,7 @@ async def test_validate_config_no_account_name():
 
     # Setup
     async with create_source(AzureBlobStorageDataSource) as source:
-        source.configuration.set_field(name="account_name", value="")
+        source.configuration.set_value(name="account_name", value="")
```

Comment:
Sorry, I completely misunderstood the need for `set_value`.

You can init source like it's done here: https://github.com/elastic/connectors-python/blob/main/tests/sources/test_jira.py#L462

```python
async with create_source(AzureBlobStorageDataSource), account_name="") as source:
    # something
```

Then you can delete `set_value` function - I'd avoid having functions for testing-only in prod code.

Reply:
@artem-shelkovnikov I looked into removing all iterations of `set_field` in favour of initialising the source with any specified values, but it is simply far too much work for this issue. I would prefer to create another tech debt issue and tackle it separately.

Instead I'd prefer to revert back to when we needed to specify `field_type` when calling `set_field` in tests for now.

Reply:
(Is it even an issue? Now we have to specify `field_type` if it isn't string when calling `set_field` in tests, that's all)